addons/binding/org.openhab.binding.omnilink/src/main/java/org/openhab/binding/omnilink/handler/ZoneHandler.java

```diff
@@ -23,6 +23,7 @@
 
 public class ZoneHandler extends AbstractOmnilinkHandler {
     private Logger logger = LoggerFactory.getLogger(ZoneHandler.class);
+    private ZoneStatus zoneStatus;
```

Comment:
I'm not sure its really needed here, this not a object that changes much, nor do we have threads locking, synchronizing or otherwise monitoring this object.  While I don't think it would do any harm (other then occupying a trivial amount of main memory) , I'm not sure we gain anything from it.